keyboards/wheatfield/blocked65/keymaps/walker/keymap.c

```diff
@@ -0,0 +1,37 @@
+#include QMK_KEYBOARD_H
+
+enum layers {
+  _QWERTY,
+  _RAISE,
+  _LOWER
+};
+
+enum keycodes {
+  QWERTY = SAFE_RANGE,
+  RAISE,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [_QWERTY] = LAYOUT(
+      KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS,  KC_EQL,  KC_BSPC,  KC_INSERT,    \
+      KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC,  KC_RBRC, KC_BSLS,  KC_DEL,  \
+      KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT,  KC_ENT,            KC_PGUP,    \
+      KC_LSFT,          KC_Z,    KC_X,    KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT, KC_UP,    KC_PGDN,      \
+      KC_LCTL, KC_LGUI, KC_LALT,                          KC_SPC,                 MO(_RAISE), MO(_LOWER),        KC_LEFT, KC_DOWN,  KC_RIGHT),
```

Comment:
```suggestion
      KC_LCTL, KC_LGUI, KC_LALT,                          KC_SPC,                 RAISE,   LOWER,   KC_LEFT, KC_DOWN,  KC_RIGHT),
```
This isn't that important, but I suggest adding `#define RAISE MO(_RAISE)` and `#define LOWER MO(_RAISE)` to the top so that you can keep the layout array visually aligned to 8 char columns.

Reply:
Fixed